src/logic/equiv/basic.lean

```diff
@@ -369,16 +369,24 @@ equiv_pempty _
 def pempty_equiv_pempty : pempty.{v} ≃ pempty.{w} :=
 equiv_pempty _
 
-/-- The `Sort` of proofs of a true proposition is equivalent to `punit`. -/
-def prop_equiv_punit {p : Prop} (h : p) : p ≃ punit :=
-⟨λ x, (), λ x, h, λ _, rfl, λ ⟨⟩, rfl⟩
-
 /-- The `Sort` of proofs of a false proposition is equivalent to `pempty`. -/
 def prop_equiv_pempty {p : Prop} (h : ¬p) : p ≃ pempty :=
 ⟨λ x, absurd x h, λ x, by cases x, λ x, absurd x h, λ x, by cases x⟩
 
-/-- `true` is equivalent to `punit`. -/
-def true_equiv_punit : true ≃ punit := prop_equiv_punit trivial
+/-- If both `α` and `β` have a unique element, then `α ≃ β`. -/
+def unique_equiv_unique (α β : Sort*) [unique α] [unique β] : α ≃ β :=
```

Comment:
This doesn't really match our naming scheme. I think the original name was better, although I think we can safely abbreviate `equiv_of_unique_of_unique` to `equiv_of_unique`

Reply:
To be clearer; I'd expect `unique_equiv_unique` to be something like
```lean
def unique_equiv_unique (α β : Sort*) (e : α ≃ β) : unique α ≃ unique β
```
(I don't think we actually need this definition, but it's what the name you chose best fits)